plugins/inputs/github/github_test.go

```diff
@@ -103,7 +103,7 @@ func TestGetTags(t *testing.T) {
 		"license":  licenseName,
 	}
 
-	require.Equal(t, true, reflect.DeepEqual(getTagsReturn, correctTagsReturn))
+	require.True(t, reflect.DeepEqual(getTagsReturn, correctTagsReturn))
```

Comment:
Hmmm shouldn't
```suggestion
	require.Equal(t, correctTagsReturn, getTagsReturn)
```
this be sufficient? IIRC `require.Equal` _does_ a deep equal...

Reply:
Changed.